tests/remote_js/test/shared.js

```diff
@@ -0,0 +1,51 @@
+var config = require('config');
+const API = require('../api2.js');
+const API3 = require('../api3.js');
+const { resetGroups } = require("../groupsSetup.js");
+
+module.exports = {
+
+	resetGroups: async () => {
+		await resetGroups();
+	},
+	
+
+	API1Setup: async () => {
+		const credentials = await API.login();
+		config.apiKey = credentials.user1.apiKey;
+		config.user2APIKey = credentials.user2.apiKey;
+		await API.useAPIVersion(1);
+		await API.userClear(config.userID);
+	},
+	API1WrapUp: async () => {
+		await API.userClear(config.userID);
+	},
```

Comment:
Let's use `Before` and `After` like Mocha, `SetUp` and `TearDown` like PHPUnit, `Start`/`Stop`, `Start`/`Shutdown`, or something else consistent and distinct.

(`Setup` is a noun, so would need to be `SetUp` to be consistent, but `SetUp` and `WrapUp` are a bit too similar.)

Reply:
I renamed them for `API#Before` and `API#After`